xunit-autolabeler-v2/ast_parser/lib/file_utils.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Unit tests for file_utils.py?

Reply:
See comment below - I'm hoping to address this via E2E tests.